Server/VikingVault.API/Controllers/UniqueEmailController.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using VikingVault.Services.Abstractions;
+using VikingVault.DataAccess.Models;
+
+
+namespace VikingVault.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UniqueEmailController : ControllerBase
+    {
+
+        private readonly IUniqueEmailService _uniqueEmailService;
+
+        public UniqueEmailController(IUniqueEmailService uniqueEmailService)
+        {
+            _uniqueEmailService = uniqueEmailService;
+        }
+
+      
+        // POST: api/UniqueEmail
+        [HttpPost]
+        public ActionResult Post([FromBody] UserEmail userEmail)
+        {
+            bool? isUnique = _uniqueEmailService.IsUniqueEmail(userEmail.email);
+
+            if (isUnique == null)
+                return StatusCode(500, "Internal Server Error");
+
+            if (isUnique == true)
+                return Ok("unique");
```

Comment:
Why don`t we return a bool response so that the frontend can implement some logic based on the response?